File: src/main/java/spring/springserver/domain/file/data/response/FileUploadResponse.java
package spring.springserver.domain.file.data.response;

public record FileUploadResponse(

        String fileUrl,

        String message
) {

    public static FileUploadResponse of(String fileUrl,
                                        String message) {

        return new FileUploadResponse(fileUrl,
                                      message);
    }
}

File: src/main/java/spring/springserver/domain/file/data/response/FileUploadResponse.kt
package spring.springserver.domain.file.data.response

data class FileUploadResponse(
    val fileUrl: String,
    val message: String
) {

    companion object {

        fun of(fileUrl: String, message: String): FileUploadResponse {
            return FileUploadResponse(
                fileUrl,
                message
            )
        }
    }
}
